app/assets/javascripts/hyrax/authority_select.es6

```diff
@@ -60,16 +58,5 @@ export default class AuthoritySelect {
  * intialize the Hyrax autocomplete with the fields that you are using
  */
 function setupAutocomplete() {
```

Comment:
It's not exported so only that class can use it. When activating the autocomplete was more involved I needed it that way, but may not anymore.